src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/StaffCategoryOrgRoleTest.java

```diff
@@ -21,7 +21,7 @@
     @Test
     void shouldApproveOrgRequestedRoleForTCW_S001() {
         //clientId check not implemented yet
-        //assignmentRequest.getRequest().setClientId("am_org_role_mapping_service");
+        assignmentRequest.getRequest().setClientId("am_org_role_mapping_service");
```

Comment:
Write more test case to support positive and negative scenarios with  the help of  byPassOrgDroolRule  flag in request.

Reply:
Done